docs/concepts/providers.md

```diff
@@ -7,41 +7,34 @@ indent: true
 
 # Providers
 
-Providers extend Crossplane to enable infrastructure resource provisioning. In
-order to provision a resource, a Custom Resource Definition (CRD) needs to be
-registered in your Kubernetes cluster and its controller should be watching the
-Custom Resources those CRDs define. Provider packages contain many Custom
-Resource Definitions and their controllers.
+Providers are Crossplane packages that bundle a set of [Managed
+Resources][managed-resources] and their respective controllers to allow
+Crossplane to provision the respective infrastructure resource.
 
 Here is the list of prominent providers:
```

Comment:
Would it make sense to make this list its own page (somewhere) rather than being under concepts? It feels a little hidden here to me.

Reply:
Certainly. Actually the list shouldn't be here in the first place it should be derived from the registry. At this point I would like to keep the list here and refactor at a later stage. Main purpose for me was to have the Terrajet providers mentioned in the docs.